### ISSUE TITLE
The rise and fall of Getting Things Done

### ISSUE THREAD

**Issue by SimonLab** [dwyl/product-ux-research#24]
Article by [Cal Newport](https://www.calnewport.com/about/): https://www.newyorker.com/tech/annals-of-technology/the-rise-and-fall-of-getting-things-done

**Comment by nelsonic** [comment]
Yeah, saw that on HN this morning: https://news.ycombinator.com/item?id=25131848 is it worth a read? 🤔 
Edit: **PDF** without ads: [The Rise and Fall of Getting Things Done _ The New Yorker.pdf](https://github.com/dwyl/product-ux-research/files/5566420/The.Rise.and.Fall.of.Getting.Things.Done._.The.New.Yorker.pdf)

**Comment by SimonLab** [comment]
Yes I found the article interesting, should have taken some notes while reading.
The article is more focused on distractions linked to "big" organisations.
The main idea I got from it, is that "productivity" tools/systems are not useful when the main issue is interruptions (zoom meetings, notifications...).
